src/LibHdb++Cassandra.h

```diff
@@ -1,16 +1,21 @@
-//=============================================================================
-//
-// file :        LibHdb++Cassandra.h
-//
-// description : Include for the LibHdb++Cassandra library.
-//
-// Author: Reynald Bourtembourg
-//
-// $Revision: $
-//
-// $Log: $
-//
-//=============================================================================
+/* Copyright (C) : 2017
```

Comment:
If we put dates in the copyright notice, I think we should probably include the previous years too.
So in this case 2014-2017